README.md

```diff
@@ -33,7 +33,14 @@ provider "tailscale" {
 ```
 
 In the `provider` block, replace `api_key` and `tailnet` with your own tailnet and API key. Alternatively, use the
-`TAILSCALE_API_KEY` and `TAILSCALE_TAILNET` environment variables.
+`TAILSCALE_API_KEY` and `TAILSCALE_TAILNET` environment variables. The api_key can also take the Terraform file function, e.g if your API key is in a file called `creds/tailscale.key`:
+
+```
+provider "tailscale" {
+  api_key = file("../creds/tailscale.key")
+  tailnet = "example.com"
```

Comment:
I think this can be `tailnet = "-"` which means "the tailnet of the API key" and avoid the error about the tailnet name not matching.